python/xpctl/sql/backend.py

```diff
@@ -215,14 +215,24 @@ def _get_filtered_metrics(allmetrics, metrics):
         else:
             return [x for x in allmetrics if x.label in metrics]
 
-    def get_results(self, task, dataset, event_type, num_exps=None, num_exps_per_config=None, metric=None, sort=None):
+    def get_results(self, task, dataset, event_type, num_exps=None, num_exps_per_config=None, metric=None, sort=None, id=None, label=None):
         session = self.Session()
         results = []
         metrics = listify(metric)
         metrics_to_add = [metrics[0]] if len(metrics) == 1 else []
         phase = self.event2phase(event_type)
-        hits = session.query(Experiment).filter(Experiment.dataset == dataset). \
-            filter(Experiment.task == task)
+        if id is not None:
```

Comment:
`id` is a builtin function in python which is getting overridden here (it also gets highlighted by a lot of editors) so most people don't use it as a variable name

Reply:
I'm aware, but I already have to use `id` in order for the CLI to do what I want, so I dont see the point in switching the var name mid stream